sandbox/public/index.html

```diff
@@ -51,7 +51,7 @@
 
       alloy("configure", {
         // edgeDomain: "konductor.int.gslb.eegw.adobedc.net",
-        // optInEnabled: true,
+        // defaultConsent: "in",
```

Comment:
Do we have a ticket to update the docs?

Reply:
Yes. I will be working on this today.

Reply:
https://jira.corp.adobe.com/browse/CORE-39951